tests/testthat/answers/model-print-output.stan

```diff
@@ -1,11 +1,11 @@
 data {
   int<lower=0> N;
-  int<lower=0,upper=1> y[N];
+  array[N] int<lower=0, upper=1> y;
 }
 parameters {
-  real<lower=0,upper=1> theta;
+  real<lower=0, upper=1> theta;
 }
 model {
-  theta ~ beta(1,1);  // uniform prior on interval 0,1
+  theta ~ beta(1, 1); // uniform prior on interval 0,1
```

Comment:
This model is used in tests for printing. I just updated this file and the .rds file.